### ISSUE TITLE
Unable to run create-ink-app --typescript project

### ISSUE THREAD

**Issue by parkerault** [vadimdemedes/ink#435]
Following the getting started guide:
```sh
user@DESKTOP-CFOTQL1:~/projects/foo$ npx create-ink-app --typescript
user@DESKTOP-CFOTQL1:~/projects/foo$ npm run start

> foo@0.0.0 start /home/user/projects/foo
> npm run build && dist/cli.js


> foo@0.0.0 build /home/user/projects/foo
> tsc

(node:19237) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
/home/user/projects/foo/dist/cli.js:2
import React from 'react';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at wrapSafe (internal/modules/cjs/loader.js:979:16)
    at Module._compile (internal/modules/cjs/loader.js:1027:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! foo@0.0.0 start: `npm run build && dist/cli.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the foo@0.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2021-05-04T21_25_50_771Z-debug.log
```

Using node version 14.16.1 lts

**Comment by Caryyon** [comment]
Getting similar issues but about `meow`
<img width="790" alt="Screen Shot 2021-05-05 at 8 22 53 PM" src="https://user-images.githubusercontent.com/9284960/117229171-14500480-ade0-11eb-8bac-df7ba7bc9c63.png">
I've made it known there as well [meow:172](https://github.com/sindresorhus/meow/issues/172)

**Comment by LitoMore** [comment]
I created a PR for fixing this issue https://github.com/vadimdemedes/create-ink-app/pull/14

**Comment by chrisvoo** [comment]
When is the PR going to be merged?

**Comment by mattdanielmurphy** [comment]
Just ran into this issue myself, for the record. Sorry if I'm spamming or something.

**Comment by iamjaredwalters** [comment]
Anything I can do to help move this along?

**Comment by RileyMShea** [comment]
I am also experiencing this on node  LTS v14

**Comment by LitoMore** [comment]
I tried to leave a message to @vadimdemedes on Twitter. And [asking `@sindresorhus` for help](https://github.com/vadimdemedes/create-ink-app/pull/14#issuecomment-879568268).

Before the problem is solved, for CommonJS modules you can apply [my changes in this PR](https://github.com/vadimdemedes/create-ink-app/pull/14/files), and rerun `npm install`.

**Comment by vadimdemedes** [comment]
Fix released in v2.1.1 thanks to @LitoMore!
